### PR TITLE
Improve DX

### PR DESCRIPTION
**Pull request by jenr24G**

### CODE REVIEW

**Review by robinbb — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,4 @@
 /target
-.idea/*
```
Comment (trevorbernard) [inline .gitignore:2]: You should leave this in here. I'm experimenting with idea RustRover

Comment (jenr24G) [reply .gitignore:2]: hey how's that going? It seems great to me except that you can't connect to a vm over ssh

Comment (trevorbernard) [reply .gitignore:2]: I quite like it. I have a long history with intellij since my background is Java

**Review by robinbb — APPROVED** [review]

**Comment by robinbb** [comment]
@trevorbernard I made the final commit, so GitHub is not counting my review, so I cannot merge. Care to give a review?

**Review by trevorbernard — APPROVED** [review]

**Pull request merged**